### PR TITLE
added Lancaster Sensorimotor Norms to predefined dictionaries

### PR DESCRIPTION
**Pull request by rbroc**
Added [this dataset](https://osf.io/7emr6/) to predefined dictionaries. 
Includes sensorimotor norms (5 senses + interoception + effectors) for ~40,000 English words.

### CODE REVIEW

**Comment by rbroc** [comment]
Note that (unlike other dictionaries) all the words in the "Word" index column are in upper case.
To overcome this, PredefinedDictionaryExtractor needs to be initialised overriding True default for case_sensitive argument. 
An alternative more general solution would be to add to init or extract method the option to preprocess dictionary index (or columns in general) before applying the extractor to a stimulus.

**Comment by tyarkoni** [comment]
Hmmm.... probably the default should actually be set to `case_sensitive=False`, as I doubt (m)any of these dictionaries have different norms for different-cased versions of the same word.

**Comment by rbroc** [comment]
default changed.

File: pliers/datasets/dictionaries.json [hunk 1]
```diff
@@ -192,5 +192,64 @@
...
+        "license": "https://creativecommons.org/licenses/by/4.0/",
+        "format": "csv",
+        "language": "english",
+        "index":"Word",
+        "columns":["Auditory.mean","Gustatory.mean","Haptic.mean","Interoceptive.mean","Olfactory.mean","Visual.mean","Foot_leg.mean","Hand_arm.mean","Head.mean","Mouth.mean","Torso.mean","Auditory.SD","Gustatory.SD","Haptic.SD","Interoceptive.SD","Olfactory.SD","Visual.SD","Foot_leg.SD","Hand_arm.SD","Head.SD","Mouth.SD","Torso.SD","Max_strength.perceptual","Minkowski3.perceptual","Exclusivity.perceptual","Dominant.perceptual","Max_strength.action","Minkowski3.action","Exclusivity.action","Dominant.action","Max_strength.sensorimotor","Minkowski3.sensorimotor","Exclusivity.sensorimotor","Dominant.sensorimotor","N_known.perceptual","List_N.perceptual","Percent_known.perceptual","N_known.action","List_N.action","Percent_known.action","Mean_age.perceptual","Mean_age.action","List#.perceptual","List#.action"],
+        "column_descriptions": [
+            "mean rating (0–5) of how strongly the concept is experienced by hearing",
+             "mean rating (0–5) of how strongly the concept is experienced by tasting",
```
Comment (adelavega) [inline pliers/datasets/dictionaries.json]:
```suggestion
            "mean rating (0–5) of how strongly the concept is experienced by tasting",
```

**Comment by rbroc** [comment]
Tests were previously failing because or 302 status code error for server request sent during `test_dicts_exist_at_url_and_initialize` in `test_datasets.py`. I've now edited the test so to allow 302 redirect status, as OSF download URLs (osf.io/file_id/download) redirect to "temporary" storage URLs (which change if file version or storage locations is updated). 
@tyarkoni @adelavega let me know if you have something against this solution.

Now though same miniconda-related failure issue as in #372 and #374.

**Comment by tyarkoni** [comment]
Great, thanks. I cleared the cache for this branch and restarted the tests, so let's see if that solves the problem.

**Comment by adelavega** [comment]
@rbroc you should merge master so the diff for this PR clears

**Comment by adelavega** [comment]
@rbroc you can merge master now to clean the diffs also

**Comment by adelavega** [comment]
Tests passed, merging.

**Pull request merged**